### PR TITLE
Feature header

### PR DESCRIPTION
**Pull request by Oksanna12**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -29,9 +39,18 @@ function App() {
       {/* Define routes */}
       <main>
         <Routes>
-          <Route path="/new" element={<NewPage />} />
-          <Route path="/" element={<Header />} />
+          {/* <Route path="/new" element={<NewPage />} /> */}
```
Comment (YuriiSkyba1) [inline src/App.tsx]: you can remove unused comments

File: src/Templates.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+[{"id":1,"title":"Wiberg Super Cure","price":"$5.41"},
```
Comment (YuriiSkyba1) [inline src/Templates.json]: you need to store this file in some assets folder or on the place where you have redux

File: src/components/FOOTER/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```
Comment (YuriiSkyba1) [inline src/components/FOOTER/Footer.jsx]: folder names need to be like Footer, Header

File: src/components/FOOTER/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import { Link } from 'react-router-dom';
+
+
+const Footer = () => {
+	let date = new Date().getFullYear();
+
+	return <footer className=''>
```
Comment (YuriiSkyba1) [inline src/components/FOOTER/Footer.jsx]: remove empty className

File: src/components/HEADER/Header.module.css [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+#login{
+	box-shadow: 5px 25px 50px 12px rgb(0 0 0 / 0.25);
+}
+#cart_mobile{
+	display: none;
+}
+
+
```
Comment (YuriiSkyba1) [inline src/components/HEADER/Header.module.css:63]: tip for future tailwind + css/scss 
and apply 
https://tailwindcss.com/docs/using-with-preprocessors

File: src/components/HEADER/NavigationLink.jsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
```
Comment (YuriiSkyba1) [inline src/components/HEADER/NavigationLink.jsx]: it's not required to import React in new versions

File: src/components/SOPPING-CART/SoppingCart.jsx [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React, {useState} from 'react';
+import style from './../HEADER/Header.module.css'
+
+const SoppingCart = () => {
+	return (
+		<div id={style.cart_icon} className='ml-16'>
+			<button className='w-9 h-9 hover:scale-110 duration-500'>
+				<img  src='https://cdn2.iconfinder.com/data/icons/Siena/48/shopping_cart%20yellow.png' ></img>
```
Comment (YuriiSkyba1) [inline src/components/SOPPING-CART/SoppingCart.jsx]: you can also add alt attribute

File: src/components/SEARCHBAR/SearchBar.jsx [hunk 8]
```diff
@@ -0,0 +1,48 @@
+import React, { useState} from 'react';
+import style from './../HEADER/Header.module.css';
```
Comment (YuriiSkyba1) [inline src/components/SEARCHBAR/SearchBar.jsx]: import headerStyles from './../HEADER/Header.module.css';

File: src/components/SEARCHBAR/SearchBar.jsx [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+	const [value, setValue] = useState("");
+	const onChange =(e) => {
+		setValue(e.target.value);
+	};
+
+	const onSearch =(searchTerm) => {
+		setValue(searchTerm);
+	}
+
+
```
Comment (YuriiSkyba1) [inline src/components/SEARCHBAR/SearchBar.jsx:14]: you can add it const searchTerm = value.toLowerCase();

File: src/components/SEARCHBAR/SearchBar.jsx [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+				<div>
+					<a href='#'>
+						<img className='absolute right-0 z-50 w-7 h-6 rounded-full pr-1 '
+							src='https://cdn2.iconfinder.com/data/icons/lightly-icons/30/search-240.png'></img>
+					</a>       
+				</div>
+			</form>
+		</div>
+		<div id={style.dropdown} className='absolute top-12 w-52 ml-3 mt-1 bg-slate-50 rounded border-grey flex flex-col'>
```
Comment (YuriiSkyba1) [inline src/components/SEARCHBAR/SearchBar.jsx]: ` <div id={headerStyles.dropdown} className='absolute top-12 w-52 ml-3 mt-1 bg-slate-50 rounded border-grey flex flex-col'>
        {data.filter(item => item.title.toLowerCase().startsWith(searchTerm) && item.title.toLowerCase() !== searchTerm)
          .map(item => (
            <div key={item.id} onClick={() => setValue(item.title)} className='cursor-pointer text-start text-base mx-1 my-1'>
              {item.title}
            </div>
          ))}
      </div>`

Comment (YuriiSkyba1) [reply src/components/SEARCHBAR/SearchBar.jsx]: you can try optimize it a bit like my example

File: src/pages/NewPage/Login.jsx [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+		}),
+
+	});
+
+	return <div className="min-h-screen">
+		<NavigationLink />
+		<div className="pt-5">
+			<div id={style.login} className='w-7/12 m-auto border-double border-4 p-1'>
+				<div className='text-2xl font-bold text-orange-950 pt-2'>Login</div>
+				<form id={style.forma} onSubmit={formik.handleSubmit} className="  text-xl space-y-2 flex justify-center flex-col m-auto w-64 h-72">
```
Comment (YuriiSkyba1) [inline src/pages/NewPage/Login.jsx]: remove here space className="

File: src/components/SEARCHBAR/SearchBar.jsx (hunk 8 above)
Comment (Oksanna12) [reply src/components/SEARCHBAR/SearchBar.jsx:2]: Should I change it everywhere or only in searchBar?

**Review by YuriiSkyba1 — APPROVED** [review]

**Pull request merged**